cmd/run.go

```diff
@@ -352,6 +353,15 @@ func newDataAvailability(c config.Config, st *state.State, etherman *etherman.Cl
 		if err != nil {
 			return nil, err
 		}
+	case string(dataavailability.NearProtocol):
+		/// TODO: Feels like there should be more configuration here, but the key store is eth specific.
+		/// Since this is a configurable sidecar, it's valid to set this when it's running, but the sequencer
```

Comment:
We'd need to create a couple configuration options for this. Whats the best approach to do that?